receiver/hostmetricsreceiver/README.md

```diff
@@ -135,83 +135,13 @@ service:
 
 #### Transition from metrics with "direction" attribute
```

Comment:
Newbie question here, but since this initiative is dropped, is there any point to still have this section at all? Doesn't it just create some confusion for people without the historical context? For the latter, changelogs and deprecation warnings should clarify it.

Reply:
I was leaving this title here for historical context with the plan to remove it when the flags are removed. I can see how it would be confusing to new users, but still wanted to leave information about the featuregates in the readme for each component, in case users out there are not reading the changelog but are reading the readme for the receivers they care about.